lib/html_utils/v6.rb

```diff
@@ -0,0 +1,46 @@
+# via: https://github.com/harvard-lil/h2o/blob/504b5e5d256c154ef50837b48fc69aa6956a9d91/lib/html_helpers.rb
+
+module HTMLUtils
+  class V6 < V5
+    EFFECTIVE_DATE = Date.new(2019, 5, 26)
+    class << self
+      def parse html
+        Nokogiri.HTML5(html)
+      end
+
+      # Tidy notes:
+      # in HTML 5 mode, Tidy doesn't add missing <ul> around bare <li> tags:
+      # https://github.com/htacg/tidy-html5/issues/525
+      # nor does it convert <ul> tags to <div> tags. In HTML 4 mode it does both of these things
```

Comment:
Becky and I weren't able to get any Tidy results in HTML4 mode even if they were in HTML4. Ex: Really old cases like these: `Case.find([12, 18, 27])`

Were you able to see old cases being processed in this way?

Reply:
This works for me. You? `Tidy.exec("<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' 'http://www.w3.org/TR/html4/strict.dtd'><html><head><title></title></head><body>#{Case.find(12).content}</body></html>")`

Reply:
Yep, Casey, that works for me too:
```
echo '<!DOCTYPE HTML PUBLIC '"'"'-//W3C//DTD HTML 4.01//EN'"'"' '"'"'http://www.w3.org/TR/html4/strict.dtd'"'"'><html><head><title></title></head><body>hi</body></html>' | node_modules/htmltidy2/bin/linux64/tidy
Info: Doctype given is "-//W3C//DTD HTML 4.01//EN"
Info: Document content looks like HTML 4.01 Transitional
```
In any case, if we end up NOT using Tidy on annotated cases, like in https://github.com/leppert/h2o/tree/migrate-annotated-cases-without-tidy-html, we don't have to think about this 😁